lightning/src/onion_message/messenger.rs

```diff
@@ -487,7 +487,13 @@ pub trait MessageRouter {
 			.into_iter()
 			.map(|ForwardNode { node_id, short_channel_id: _ }| node_id)
 			.collect();
-		self.create_blinded_paths(recipient, context, peers, secp_ctx)
+
+		// This parameter is a placeholder. This function is removed in the subsequent commits.
```

Comment:
Doesn't look like it was removed.

Reply:
Removed the comment, and the function in [pr3246.06](https://github.com/shaavan/rust-lightning/commits/pr3246.06)
Thanks for the pointer!